ReactWindows/ReactNative/UIManager/LayoutShadowNode.cs

```diff
@@ -165,7 +168,7 @@ public void SetMargins(int index, float margin)
             ViewProps.PaddingTop,
             ViewProps.PaddingBottom,
             DefaultSingle = Undefined)]
-        public void SetPaddings(int index, float padding)
+        public virtual void SetPaddings(int index, float padding)
```

Comment:
If padding is set both for TextBlock  and CSSNode the padding on bottom is set twice.